### DISCUSSION TITLE
Brave

### DISCUSSION THREAD

**Discussion by JuliaAgostino** [mayrinck/FOSSonMicrosoftStore#10] [Requests] [answered]
Brave website: https://brave.com/

Brave privacy policy: https://brave.com/privacy/browser/

Logo: https://brave.com/static-assets/images/brave-logo-dark.svg

Description: "Brave is a free and open-source web browser developed by Brave Software, Inc. based on the Chromium web browser. Brave is a privacy focused browser, which distinguishes itself from other browsers by automatically blocking online advertisements and website trackers in its default settings. It also provides users the ability to turn on optional ads that pay users for their attention in the form of Basic Attention Tokens (BAT) cryptocurrency. Users can then send contributions to websites and content creators along with the ability to keep the cryptocurrency they earned. Brave Software's headquarters are located in San Francisco, CA.

As of May 2021, Brave has 32.4 million active monthly users and a network of 1.2 million content creators."

**Comment by mayrinck** [comment, marked as answer]
Hmmmmm... Brave might be tough to add, but maybe it works. I will check it out later if Firefox succeeds.
Thank you for submitting.
